buildchain/buildchain/image.py

```diff
@@ -80,9 +80,13 @@ def show() -> str:
             path=utils.build_relpath(constants.ISO_IMAGE_ROOT)
         )
 
-    task = targets.Target(task_dep=['_image_pull', '_image_build']).basic_task
+    task = targets.Target(
+        task_dep=['check_for:hardlink', '_image_pull', '_image_build']
+        ).basic_task
```

Comment:
This one should be unindented:

```suggestion
    ).basic_task
```